eos/maths/gegenbauer-polynomial.cc

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) 2023 Viktor Kuschke
+ *
+ * This file is part of the EOS project. EOS is free software;
+ * you can redistribute it and/or modify it under the terms of the GNU General
+ * Public License version 2, as published by the Free Software Foundation.
+ *
+ * EOS is distributed in the hope that it will be useful, but WITHOUT ANY
+ * WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more
+ * details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * this program; if not, write to the Free Software Foundation, Inc., 59 Temple
+ * Place, Suite 330, Boston, MA  02111-1307  USA
+ */
+
+#include <eos/maths/power-of.hh>
+#include <eos/maths/gegenbauer-polynomial.hh>
+
+#include <cmath>
+#include <vector>
+
+namespace eos
+{
+    std::vector<double> GegenbauerPolynomial::calculate_coefficients() const
+    {
+        const double epsilon = 1e-8;
+        double entry;
+        std::vector<double> result (half_order + 1);
```

Comment:
Coding style: please remove the whitespace between the identifier and the call to the constructor.